catalog/spatial/wps/spatial-wps-endpoint/src/main/java/org/codice/ddf/spatial/ogc/wps/process/endpoint/WpsEndpoint.java

```diff
@@ -0,0 +1,514 @@
+
+    /**
+     * @param getStatus
+     * @return
+     * @throws WpsException
+     */
+    @Override
+    public StatusInfo getStatus(GetStatus getStatus) {
+        return getStatus(getStatus.getJobID());
+    }
+
+    /**
+     * @param jobId
+     * @return
+     * @throws WpsException
+     */
+    @Override
+    public Result getResult(String jobId) {
+        ProcessResult processResult = processMonitors.stream()
+                .map(am -> {
+                    try {
+                        return am.getRequestResult(jobId);
+                    } catch (RuntimeException e) {
+                        LOGGER.debug("Error monitoring requests", e);
+                        return null;
+                    }
+                })
+                .filter(Objects::nonNull)
+                .findFirst()
+                .orElseThrow(() -> new WpsException(
+                        "The JobID from the request does not match any of the Jobs running on this server",
+                        "NoSuchJob",
+                        jobId));
+        return wpsObjectConverter.createResult(processResult);
+    }
+
+    /**
+     * @param getResult
+     * @return
+     * @throws WpsException
+     */
+    @Override
+    public Result getResult(GetResult getResult) {
+        return getResult(getResult.getJobID());
+    }
+
+    /**
+     * @param jobId
+     * @return
+     * @throws WpsException
+     */
+    @Override
+    public StatusInfo dismiss(String jobId) {
+        ProcessStatus status = processMonitors.stream()
+                .map(am -> {
+                    try {
+                        return am.dismissRequest(jobId);
+                    } catch (RuntimeException e) {
+                        LOGGER.debug("Error monitoring requests", e);
+                        return null;
+                    }
+                })
+                .filter(Objects::nonNull)
+                .findFirst()
+                .orElseThrow(() -> new WpsException(
+                        "The JobID from the request does not match any of the Jobs running on this server",
+                        "NoSuchJob",
+                        jobId));
+        return wpsObjectConverter.createStatusInfo(status);
+    }
+
+    /**
+     * @param dismiss
+     * @return
+     * @throws WpsException
+     */
+    @Override
+    public StatusInfo dismiss(Dismiss dismiss) {
+        return dismiss(dismiss.getJobID());
+    }
+
+    /**
+     * @param processId
+     * @return
+     * @throws WpsException
+     */
+    private Process getProcess(String processId) {
+        for (ProcessRepository algorithmRepository : processRepositories) {
+            try {
+                Process algorithm = algorithmRepository.getProcess(processId);
+                if (algorithm != null) {
+                    return algorithm;
+                }
+            } catch (RuntimeException e) {
+                LOGGER.debug("Error retrieving process from repository {}", processId, e);
```

Comment:
As the log message is currently written it looks like the last part of the message is the repository name, not the process name.